### ISSUE TITLE
Provide simpler overloads of ScriptServicesBuilder.ctor()

### ISSUE THREAD

**Issue by adamralph** [scriptcs/scriptcs#976]
Based on feedback from @bitbonk regarding https://gist.github.com/bitbonk/aea4cb41a21b7d666e80#file-program-cs-L15

A hosting consumer may have no interest in the two arguments which currently have to be supplied, so we should probably provide at least an overload which expects neither.

**Comment by glennblock** [comment]
I agree we should at least evaluate doing some further simplification.

Having to create the console and loggers is an annoyance.

As to loading of packages, we didn't do this by default in order to make hosts explicitly choose. Same with modules. 

We could look at having a LoadPackages method on the builder that has an optional (InitializeScriptPacks) param as an example.

**Comment by bitbonk** [comment]
I'd even go as far and say that the whole

    var scriptcs = new ScriptServicesBuilder(new ScriptConsole(), new NoOpLogger())
                .Repl()
                .ScriptEngine<RoslynScriptEngine>()
                .Build();

could be hidden behind a facade. Just give me resonable defaults for common scenarios.

**Comment by filipw** [comment]
the ctor parameters are an annoyance, I agree - they should use defaults. 

The rest, I am not sure - really the only thing you must specify is engine - it already has all the rest as defaults - i.e. `Repl` takes true/false so you only call it when you want a REPL, default is no REPL. I guess it could be changed so that you don't need to call `Build`

At the end of the day, it's similar to configuring for example ServiceBus:

```
    var bus = Configure.With()
        .DefaultBuilder()
        .Log4Net()
        .MsmqSubscriptionStorage()
        .DefiningCommandsAs(t => t.Namespace.StartsWith("Commands"))
        .UseTransport<Msmq>()
        .UnicastBus()
        .DisableTimeoutManager()
        .SendOnly();
```
